internal/servers/controller/handlers/authmethods/authmethod_service.go

```diff
@@ -814,6 +815,12 @@ func validateCreateRequest(req *pbs.CreateAuthMethodRequest) error {
 						}
 					}
 				}
+				if len(attrs.GetAccountClaimMaps()) > 0 {
+					_, err := oidc.ParseAccountClaimMaps(attrs.GetAccountClaimMaps()...)
```

Comment:
Please confirm that multiple kv pairs do not map to the same ToClaim since the errors returned here can be nicer than when returned by the DB.  Both here and in update.

Reply:
ty!